README.md

```diff
@@ -51,9 +51,9 @@ With the above steps completed, you can start a preview server by running this c
 yarn dev
 ```
 
-This starts a preview server on your machine, and watches all files for changes. Open http://localhost:8080/documentation/ to view the documentation.
+This starts a preview server on your machine, and watches all files for changes. Open http://localhost:8080/docs/ to view the documentation.
 
- > Note: if you try to visit the root, you will get a 404 page.  You must visit a path corresponding to a [directory under `vuepress-site`](https://github.com/okta/okta-developer-docs/tree/master/packages/%40okta/vuepress-site), like `/documentation`.
```

Comment:
It's actually `/docs/` with the terminating slash. If you can fix this, I'll get this merged ASAP.

Reply:
incorporated the changes.